examples/LLM/llama2/chat_app/Readme.md

```diff
@@ -0,0 +1,125 @@
+
```

Comment:
mention this on the main README as well as a way for people to deploy an end to end chatbot

Reply:
added